polygeist/tools/cgeist/Lib/clang-mlir.cc

```diff
@@ -3664,7 +3663,7 @@ ValueCategory MLIRScanner::VisitCastExpr(CastExpr *E) {
     }
     assert(se.val);
     auto Derived =
-         (E->isLValue() || E->isXValue())
+        (E->isLValue() || E->isXValue())
```

Comment:
Unrelated change?

Reply:
Right, just clang-format. Let's have it in, so we can clang-format the whole file in the future.

Reply:
If we are planning to clang-format the file, I would prefer to be a separate patch.  Let's not pollute git history.